mewarpx/examples/thermionic_diode_rz_cylinder.py

```diff
@@ -0,0 +1,194 @@
+import argparse
+import sys
+
+from mewarpx.utils_store import util as mwxutil
+
+mwxutil.init_libwarpx(ndim=2, rz=True)
+
+import numpy as np
+from pywarpx import picmi
+
+from mewarpx import assemblies, diags, emission, mespecies, runinfo
+from mewarpx.mwxrun import mwxrun
+
+
+class VacuumTEC_Cylinder(object):
```

Comment:
```suggestion
class CylinderVacuumTEC(object):
```
Underscores in python classes aren't standard, so if you don't like `TECCylinder` this seems like a good alternative.